Scripts/_FullTestSuite_.bv.lua

```diff
@@ -57,7 +57,8 @@ tests= {
     RunScriptWrapper('scripts/streamingimagetest.bv.luac'),
     RunScriptWrapper('scripts/parallaxtest.bv.luac'),
     RunScriptWrapper('scripts/parallaxdepthartifacts.bv.luac'),
-    RunScriptWrapper('scripts/checkerboardtest.bv.luac'),
+    -- Disable checkerboard pipeline as it's quite out of date and currently a maintenance burden
+    -- RunScriptWrapper('scripts/checkerboardtest.bv.luac'),
```

Comment:
Is there a plan to fix it, or to remove it entirely? It seems we need a GHI to follow up on this.

Reply:
Will discuss with the owner of this component when they return.